docs/elements.md

```diff
@@ -780,6 +782,42 @@ In the submission panel, a `pl-matrix-input` element displays either the submitt
 
 -----
 
+### `pl-rich-text-editor` element
```

Comment:
Assuming you're looking into `pl-matching` (#3670) at some point in the near future, I wonder if it may be better to have a single news item with recent additions to elements. Maybe also include the `feedback` argument in `pl-multiple-choice` (#4656). And (if you're open to review them) `pl-xss-safe` in #3475 and `pl-feedback` in #3865 ?

Reply:
Yes, sorry, I feel really bad about holding up all these other excellent element improvements for no good reason. I'm reviewing now!

Reply:
(Feel free to merge this PR whenever you like.)